timestamp.md

```diff
@@ -9,7 +9,10 @@ The `TIMESTAMP` [data type](data-types.html) stores a date and time pair in UTC.
 
 ## Aliases
 
-In CockroachDB, `TIMESTAMPZ` is an alias for `TIMESTAMP`.
+In CockroachDB, the following are aliases for `TIMESTAMP`:
+
+- `TIMESTAMPZ` 
```

Comment:
I don't think that's true. We have two timestamp types:
* `TIMESTAMP WITHOUT TIME ZONE`, which may be (and usually is) abbreviated `TIMESTAMP`
* `TIMESTAMP WITH TIME ZONE`, which may be (less commonly) abbreviated `TIMESTAMP TZ`